app/assets/javascripts/components/overview/details.cjsx

```diff
@@ -54,6 +64,58 @@ Details = React.createClass(
         </fieldset>
       )
 
+    if @props.course.expected_students
+      expected_students = (
+        <fieldset>
+          <TextInput
+            onChange={@updateDetails}
+            value={@props.course.expected_students}
+            value_key='expected_students'
+            editable={@props.editable}
+            type='number'
+            label={CourseUtils.i18n('expected_students', @props.course.string_prefix)}
+          />
+        </fieldset>
+      )
+
+    # TODO: course.timeline?
+    if @props.course.timeline_start || @props.course.timeline_end
```

Comment:
This checks for either one, but moment code below expects both timeline_start and timeline_end to be defined. It's probably not functionally relevant, as timeline start and end come from the same json data and should always be present. But it would be clearer to either just check for one, or check for both with &&.